scripts/scripts.js

```diff
@@ -48,6 +48,30 @@ export function replaceLinksWithEmbed(block) {
       twitterLink.replaceWith(embed);
     }
   });
+
+  // handling legal privacy iframes
+  block.querySelectorAll('a[href*="view.ceros.com"]').forEach((legalEmbed) => {
```

Comment:
On landing pages, we have embeds that are iframes with highly animated content from ceros so it could clash with this code WDYT ?

<img width="437" alt="Screenshot 2023-09-14 at 09 32 22" src="https://github.com/headwirecom/helix-sportsmagazine/assets/6012307/b453f712-e0e0-4add-aae5-8ab27e070711">